GOVERNANCE.md

```diff
@@ -6,50 +6,27 @@ This document defines the project governance for Kubeapps.
 
 **Kubeapps**, an open-source project, is committed to building an open, inclusive, productive and self-governing open source community involved to simplifying how applications are deployed and managed in Kubernetes clusters. The community is governed by this document to define how the community should work together to achieve this goal.
 
-## Code Repositories
+## Code of Conduct
```

Comment:
Some minor typos:

Not here, but above:

`involved TO simplifying`  -> `involved in simplifying`

And at the "credits" section:

`Sections of this documentS have .... ` --> `Sections of this document have .... `